src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1441,6 +1458,22 @@ ProviderTypes.Spotify or ProviderTypes.StackExchange or ProviderTypes.Zoom
                 // Typeform returns the username as a custom "alias" node:
                 ProviderTypes.Typeform => (string?) context.UserInfoResponse?["alias"],
 
+                // VK ID returns the username as a custom "user/first_name" and "user/last_name" nodes
+                // (https://id.vk.com/about/business/go/docs/en/vkid/latest/vk-id/connection/api-integration/api-description#App-access-to-user-data):
+                ProviderTypes.VkId 
+                    when context.UserInfoResponse?["user"]?["first_name"] is not null &&
+                         context.UserInfoResponse?["user"]?["last_name"] is not null
+                    => $"{(string?) context.UserInfoResponse?["user"]?["first_name"]} {(string?) context.UserInfoResponse?["user"]?["last_name"]}",
+
+                // Yandex returns standard "login" node, if scope has "info" premission we have a set of names (first_name, last_name,
+                // display_name, real_name). (https://yandex.ru/dev/id/doc/en/user-information#name-access):
+                ProviderTypes.Yandex
+                    when context.UserInfoResponse?["login"] is not null => (string?) context.UserInfoResponse?["login"],
```

Comment:
Hum, I'm not super comfortable using different value sources for the same claim. Let's just use `display_name` here? (in that case, there's already a case for `display_name` earlier in the handler).